libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectUtils.kt

```diff
@@ -1,36 +1,39 @@
-@file:JvmName("RouteAlertUtils")
+@file:JvmName("RoadObjectUtils")
 
-package com.mapbox.navigation.base.trip.model.alert
+package com.mapbox.navigation.base.trip.model.roadobject
 
 import androidx.annotation.IntRange
 import com.mapbox.api.directions.v5.models.DirectionsRoute
 import com.mapbox.api.directions.v5.models.RouteOptions
 import com.mapbox.core.constants.Constants
 import com.mapbox.geojson.LineString
+import com.mapbox.navigation.base.utils.ifNonNull
 
 /**
- * Returns a [LineString] geometry of the alert.
+ * Returns a [LineString] geometry of the road object.
  *
- * @param directionsRoute the original route that this alert is a part of.
+ * @param directionsRoute the original route that this object is a part of.
  * @param precision precision of the polyline encoding.
  * Possible values are [Constants.PRECISION_5] and [Constants.PRECISION_6].
  * See [RouteOptions.Builder.geometries].
  */
-fun RouteAlertGeometry.toLineString(
+fun RoadObjectGeometry.toLineString(
     directionsRoute: DirectionsRoute,
     @IntRange(from = 5, to = 6) precision: Int
-): LineString {
+): LineString? {
```

Comment:
Can this be `null`? 🤔 

https://github.com/mapbox/mapbox-java/blob/f16813f3489269011ebacfce8a56fa6b5b3279f4/services-geojson/src/main/java/com/mapbox/geojson/LineString.java#L144-L154

Reply:
yes it can, it depends on 
`fun RoadObjectGeometry.toLineString(routeLineString: LineString): LineString?`
and inside we have
` ifNonNull(startGeometryIndex, endGeometryIndex)`

so, both functions depend on `nullable` `startGeometryIndex, endGeometryIndex`